src/classes/selectors/BoutiqueResourceSelector.cls

```diff
@@ -0,0 +1,13 @@
+/**
+ * Created by Mariya on 14.11.2022.
+ */
+
+public with sharing class BoutiqueResourceSelector {
+
+    public static List<Boutique_Resource__c> getBoutiqueResourceByContactId(Set<Id> contactId){
```

Comment:
MINOR
getBoutiqueResourceByContactId -> getBoutiqueResourcesByContactIds as input param is collection, not single Id, also method returns a collection, so naming must be in the plural form

Spaces are not needed for return.

Reply:
got it, thanks